CMakeLists.txt

```diff
@@ -501,10 +515,12 @@ if(FREEBSD OR MACOS)
 endif()
 
 # openssl/crypto
-set(ENABLE_OPENSSL True)
 pkg_check_modules(OPENSSL openssl)
 
-if(NOT OPENSSL_FOUND)
+# wolfssl
+pkg_check_modules(WOLFSSL wolfssl)
```

Comment:
We should be skipping this check unless OpenSSL was not found _or_ the user asked to force usage of WolfSSL.